terraform/lang/security/ec2-imdsv1-optional.tf

```diff
@@ -0,0 +1,8 @@
+# ruleid: ec2-imdsv1-optional
+resource "aws_instance" "test-instance" {
+  ami = "ami-0d5eff06f840b45e9"
+  metadata_options {
+    http_endpoint = "enabled"
+    http_tokens = "optional"
```

Comment:
I think you'll need to move the `# ruleid: ...` comment above this line.

Also, can you add a true negative test as well? Similarly you can use the [`# ok: ...` syntax](https://semgrep.dev/docs/writing-rules/testing-rules/).